plugins/PerObjectSettingsTool/ToolBarButton.qml

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2021 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.1
+import QtQuick.Controls 2.1
+
+import Cura 1.0 as Cura
+import UM 1.5 as UM
+
+ToolButton
```

Comment:
I don't think you should use a ToolButton here. A regular button should do (since you're not using it to display an Icon)

Reply:
https://github.com/Ultimaker/Cura/pull/11336/commits/ad3832791c184489b2e4bf9ac578621959359b90